test/functional/ui/screen_basic_spec.lua

```diff
@@ -243,6 +241,30 @@ describe('Screen', function()
         -- INSERT --                                         |
       ]])
     end)
+
+    -- https://code.google.com/p/vim/issues/detail?id=339
+    it([[setting 'ruler' doesn't reset the preferred column]], function()
```

Comment:
The `describe` block is labelled 'insert mode' but this test isn't about insert mode. Looking at the other `describe` blocks in this file perhaps moving the test case into a `describe('normal mode', ...` block of its own seems fair.